README.md

```diff
@@ -178,11 +178,12 @@ docker run --rm -it \
 -v [ASYNCAPI SPEC FILE LOCATION]:/app/asyncapi.yml \
 -v [GENERATED FILES LOCATION]:/app/output \
 asyncapi/generator [COMMAND HERE]
+
 # Example that you can run inside generator directory after cloning this repository. First you specify mount in location of your AsyncAPI specification file and then you mount in directory where generation result should be saved.
 docker run --rm -it \
 -v ${PWD}/test/docs/dummy.yml:/app/asyncapi.yml \
 -v ${PWD}/output:/app/output \
-asyncapi/generator -o ./output asyncapi.yml @asyncapi/html-template --force-write
+asyncapi/generator -o /app/output /app/asyncapi.yml @asyncapi/html-template --force-write
```

Comment:
you think it is better to do `/app/asyncapi.yml` instead of `asyncapi.yml` in docs? will it be less confusing? both ways work as `app` is a workdir

Reply:
Yes, I think it makes the example a little bit clearer. You see immediately that the volumes you just mounted in `/app/asyncapi.yml` and `/app/output/` are used directly in the command you pass to the CLI.
Also, the docker image workdir can only be seen if a user actually looks at the `Dockerfile`. So in my opinion, this approach makes it a bit easier for docker beginners who just want to run the thing but don't care about the details.